pyproject.toml

```diff
@@ -11,7 +11,7 @@ keywords = ["parquet-tools", "parquet"]
 exclude = ["parquet_tools/parquet.thrift", "parquet_tools/README.md"]
 
 [tool.poetry.dependencies]
-python = ">=3.8"
+python = ">=3.9"
```

Comment:
Could you explain why you updated this. I know Python 3.8 EOL will come soon. But I'm not sure that it's good to update python version in this PR.

Reply:
Because `poetry` yelled at me in your GHA, see https://github.com/ktrueda/parquet-tools/actions/runs/8312677506/job/22747801180#step:5:145

Reply:
I can revert it if you like